### PR TITLE
Mdd seo doc fixing

### PR DESCRIPTION
**Pull request by mddifilippo89**
Bubble
Pie
Donut
FC
Spline

### CODE REVIEW

File: doc/en/components/charts/types/bubble-chart.md [hunk 1]
```diff
@@ -20,8 +22,7 @@ $PlatformShort$ Bubble Chart is a group of series that show markers with variabl
 
 ## $PlatformShort$ Bubble Chart with Single Series
 
-The Bubble Chart contains many of the characteristics of the Scatter Point Chart but with the option to have various radius scale sizes.
-
+The Bubble Chart contains many of the characteristics of the [Scatter Marker Chart](scatter-chart.md) but with the option to have various radius scale sizes.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/bubble-chart.md:25]: append "#$PlatformShort$-scatter-marker-chart" after ".md" to make this a link to a section of that topic

File: doc/en/components/charts/types/bubble-chart.md [hunk 2]
```diff
@@ -33,7 +34,7 @@ The Bubble Chart contains many of the characteristics of the Scatter Point Chart
 
 ## $PlatformShort$ Bubble Chart with Multiple Series
 
-The Bubble Chart contains many of the characteristics of the Scatter Point Chart but with the option to have various radius scale sizes.
+The Bubble Chart contains many of the characteristics of the [Scatter Marker Chart](scatter-chart.md) but with the option to have various radius scale sizes.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/bubble-chart.md:37]: append "#$PlatformShort$-scatter-marker-chart" after ".md" to make this a link to a section of that topic

File: doc/en/components/charts/types/bubble-chart.md [hunk 3]
```diff
@@ -48,7 +49,7 @@ In this example, the Bubble Chart is comparing the total births and deaths betwe
 
 ## $PlatformShort$ Bubble Chart Styling
 
-The Bubble Chart contains many of the characteristics of the Scatter Point Chart but with the option to have various radius scale sizes and fill scale colors.
+The Bubble Chart contains many of the characteristics of the [Scatter Marker Chart](scatter-chart.md) but with the option to have various radius scale sizes and fill scale colors.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/bubble-chart.md:52]: append "#$PlatformShort$-scatter-marker-chart" after ".md" to make this a link to a section of that topic

File: doc/en/components/charts/types/bubble-chart.md [hunk 4]
```diff
@@ -62,9 +63,13 @@ In this example, the Bubble Chart is comparing the total births and deaths betwe
...
 - [Scatter Chart](scatter-chart.md)
 - [Shape Chart](shape-chart.md)
+
+
+## Additional Resources
+The following table lists API members mentioned in the above sections:
+
+Chart Type       | Control Name   | API Members
+-----------------|----------------|------------
+Scatter Marker   | `XamDataChart`     | `ScatterSeries`
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/bubble-chart.md:75]: missing API for BubbleSeries

File: doc/en/components/charts/types/donut-chart.md [hunk 5]
```diff
@@ -61,3 +61,9 @@ It is possible to have a multiple ring display in the $PlatformShort$ Donut Char
 ## Additional Resources
 - [Pie Chart](pie-chart.md)
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/donut-chart.md:62]: add links to topics: radial-chart, polar-chart

File: doc/en/components/charts/types/pie-chart.md [hunk 6]
```diff
@@ -2,12 +2,27 @@
...
+'LegendLabelMemberPath',
+'OthersCategoryThreshold',
+'OthersCategoryType',
+'SelectedItems',
+'SelectionMode',
+'RadiusFactor',
+'StartAngle',
+
+
+ ]
```
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/pie-chart.md:18]: remove property name from the mentionedTypes array. We can list only names of classes in the mentionedTypes array

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/types/bubble-chart.md (hunk 1 above)
Comment (mddifilippo89) [reply doc/en/components/charts/types/bubble-chart.md:25]: pending

File: doc/en/components/charts/types/bubble-chart.md (hunk 2 above)
Comment (mddifilippo89) [reply doc/en/components/charts/types/bubble-chart.md:37]: pending

File: doc/en/components/charts/types/bubble-chart.md (hunk 3 above)
Comment (mddifilippo89) [reply doc/en/components/charts/types/bubble-chart.md:52]: pending

File: doc/en/components/charts/types/bubble-chart.md (hunk 4 above)
Comment (mddifilippo89) [reply doc/en/components/charts/types/bubble-chart.md:75]: pending

File: doc/en/components/charts/types/donut-chart.md [hunk 7]
```diff
@@ -2,12 +2,13 @@
...
 _keywords: $PlatformShort$ Charts, Donut Chart, Donut Chart, Infragistics
-mentionedTypes: ["XamDoughnutChart"]
+mentionedTypes: ["XamDoughnutChart", "DoughnutChart"]
 ---
 # $PlatformShort$ Donut Chart
 
-$PlatformShort$ Donut Chart is similar to the Pie chart, proportionally illustrating the occurrences of a variable. The donut chart can display multiple variables in concentric rings, and provides built-in support for visualizing hierarchical data. The rings are capable of being bound to a different data item, or they can share a common data source.
+$PlatformShort$ Donut Chart is similar to the [Pie Chart](pie-chart.md), proportionally illustrating the occurrences of a variable. The donut chart can display multiple variables in concentric rings, and provides built-in support for visualizing hierarchical data. The rings are capable of being bound to a different data item, or they can share a common data source.
 
+You can create this type of chart with the `XamDoughnutChart` control by binding your data as shown in the example below.
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/donut-chart.md:11]: pending

File: doc/en/components/charts/types/stock-chart.md [hunk 8]
```diff
@@ -2,12 +2,14 @@
...
 _keywords: $PlatformShort$ Charts, Stock Chart, Infragistics
-mentionedTypes: ["XamFinancialChart"]
+mentionedTypes: ["XamFinancialChart", `ChartType`, `CrosshairsSnapToData`, `IndicatorTypes`, `ZoomSliderType`]
 ---
 # $PlatformShort$ Stock Chart
 
 $PlatformShort$ Stock Chart, sometimes referred to as $PlatformShort$ Candlestick Chart, is a composite visualization that renders stock ticker data, or price data in an interactive time-series display. Stock ticker represents the company data (a ticker symbol, usually 1 to 5 characters) that you are analyzing. Stock charts show stock prices for a ticker over time in a Time Series X-Axis. Stock charts also show information for a company’s ticker data like Open Price, High Price, Low Price and Close Price (OHLC) for each time-period. The $PlatformShort$ Stock chart offers multiple ways in which the data can then be visualized and interpreted, including display modes for price and volume and a host of Stock indicators.
 
+You can create this type of chart with the `XamFinancialChart` control by binding your data and setting `ChartType` property to `Line` value, as shown in the example below.
+
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/stock-chart.md:12]: pending

File: doc/en/components/charts/types/spline-chart.md [hunk 9]
```diff
@@ -2,11 +2,13 @@
...
+$PlatformShort$ Spline Chart belongs to a group of Category Series and it is rendered using a collection of points connected by smooth curves of spline.  Values are represented on the y-axis and categories are displayed on the x-axis. Spline Chart emphasizes the amount of change over a period of time or compares multiple items as well as the relationship of parts to a whole by displaying the total of the plotted values. The Spline Chart is identical to the [Line Chart](line-chart.md) in all aspects except that line connecting data points has spline interpolation and smoothing for improved presentation of data.
+
+You can create this type of chart in `XamCategoryChart` control by binding your data and setting `ChartType` property to `Spline` value, as shown in the example below.
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/spline-chart.md]: ignore, Andrew got to this first and I got his changes which is already merged to vnext

File: doc/en/components/charts/types/pie-chart.md [hunk 10]
```diff
@@ -2,12 +2,27 @@
...
 ---
 # $PlatformShort$ Pie Chart
 
 $PlatformShort$ Pie Chart, or Pie Graph, is a part-to-whole chart that shows how categories (parts) of a data set add up to a total (whole) value. Categories are rendered as sections in a circular, or pie-shaped graph. Each section, or pie slice, has an arc length proportional to its underlying data value. Categories are shown in proportion to other categories based on their value percentage to the total value being analyzed, as parts of 100 or 100%.
 
+You can create this type of chart with the `XamPieChart` control by binding your data as shown in the example below.
+
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/pie-chart.md:25]: pending

File: doc/en/components/charts/types/bubble-chart.md [hunk 11]
```diff
@@ -2,12 +2,14 @@
...
 _keywords: $PlatformShort$ Charts, Bubble Chart, Infragistics
-mentionedTypes: ["XamDataChart", "BubbleSeries"]
+mentionedTypes: ["XamDataChart", "BubbleSeries", "ScatterSeries"]
 ---
 # $PlatformShort$ Bubble Chart
 
 $PlatformShort$ Bubble Chart is a group of series that show markers with variable scaling to represent the relationship among items in several distinct series of data or to plot data items using x and y coordinates. The x and y coordinates of the data point are determined by two numeric data columns. The Bubble Series draw attention to uneven intervals or clusters of data. These types of series are often used to plot scientific data, and can highlight the deviation of collected data from predicted results. Bubble series can also be used to organize data chronologically (even if the data is not in chronological order).
 
+You can create this type of chart in `XamDataChart` control by binding your data to the `BubbleSeries`, as shown in the example below.
+
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/bubble-chart.md:12]: pending

File: doc/en/components/charts/types/donut-chart.md (hunk 5 above)
Comment (HUSSAR-mtrela) [inline doc/en/components/charts/types/donut-chart.md:62]: add links to topics: radial-chart, polar-chart

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: doc/en/components/charts/types/stock-chart.md [hunk 12]
```diff
@@ -294,14 +296,8 @@ In this example, the stock chart is plotting revenue for United States.
...
-- `ChartType`
-- `CrosshairsSnapToData`
-- `IndicatorTypes`
-- `ZoomSliderType`
-- `FinancialChart`
+The following table lists API members mentioned in the above sections:
+
+Chart Type       | Control Name   | API Members
+-----------------|----------------|------------
+Stock Chart      | `XamFinancialChart`     | `FinancialChart`
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/stock-chart.md:303]: pending

File: doc/en/components/charts/types/pie-chart.md (hunk 6 above)
Comment (mddifilippo89) [reply doc/en/components/charts/types/pie-chart.md]: pending

File: doc/en/components/charts/types/pie-chart.md [hunk 13]
```diff
@@ -464,7 +479,7 @@ Once our pie chart is created, we may want to make some further styling customiz
 
 ## $PlatformShort$ Radial Pie Chart
 
-The Radial Pie Chart belongs to a group of radial charts and uses pie slices that extend from the center of chart towards locations of data points. This chart type takes concepts of categorizing multiple series of data points and wraps them around a circular axis rather than stretching data points along a horizontal line.
+The [Radial Pie Chart](radial-chart.md) belongs to a group of radial charts and uses pie slices that extend from the center of chart towards locations of data points. This chart type takes concepts of categorizing multiple series of data points and wraps them around a circular axis rather than stretching data points along a horizontal line.
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/pie-chart.md:469]: pending

File: doc/en/components/charts/types/pie-chart.md [hunk 14]
```diff
@@ -477,13 +492,12 @@ The Radial Pie Chart belongs to a group of radial charts and uses pie slices tha
 
 ## Additional Resources
 - [Donut Chart](donut-chart.md)
+- [Radial Chart](radial-chart.md)
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/pie-chart.md:482]: pending

File: doc/en/components/charts/types/pie-chart.md [hunk 15]
```diff
@@ -477,13 +492,12 @@ The Radial Pie Chart belongs to a group of radial charts and uses pie slices tha
...
-- `LegendLabelMemberPath`
-- `OthersCategoryThreshold`
-- `OthersCategoryType`
-- `SelectionMode`
-- `XamPieChart`
\ No newline at end of file
+Chart Type       | Control Name   | API Members
+-----------------|----------------|------------
+Pie Chart      | `XamPieChart`     | `PieChart`
+Radial Pie Chart | `XamDataChart` | `RadialPieSeries`
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/pie-chart.md:490]: pending

File: doc/en/components/charts/types/spline-chart.md [hunk 16]
```diff
@@ -84,10 +88,15 @@ The Stacked 100 Spline Chart is identical to the Stacked Spline Chart in all asp
 <div class="divider--half"></div>
 
 ## Additional Resources
-- [Annotations](../features/chart-annotations.md)
-- [Highlighting](../features/chart-highlighting.md)
-- [Markers](../features/chart-markers.md)
-- [Tooltips](../features/chart-tooltips.md)
-- [Trendlines](../features/chart-trendlines.md)
 - [Line Chart](line-chart.md)
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/spline-chart.md]: already merged

File: doc/en/components/charts/types/stock-chart.md [hunk 17]
```diff
@@ -2,12 +2,14 @@
 title: $PlatformShort$ Stock/Financial Charts | $ProductName$
 _description: The $ProductName$ Stock Chart is a composite visualization that renders stock ticker data, or price data in an interactive time-series display. Try for FREE.
 _keywords: $PlatformShort$ Charts, Stock Chart, Infragistics
-mentionedTypes: ["XamFinancialChart"]
+mentionedTypes: ["XamFinancialChart", `ChartType`, `CrosshairsSnapToData`, `IndicatorTypes`, `ZoomSliderType`]
 ---
```
[the remark this answers is not in the document]
Comment (mddifilippo89) [reply doc/en/components/charts/types/stock-chart.md:6]: pending

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**